src/raiden_libs/blockchain.py

```diff
@@ -118,31 +119,39 @@ def parse_token_network_event(event: dict) -> Optional[Event]:
     event_name = event["event"]
 
     common_infos = dict(
-        token_network_address=decode_hex(event["address"]),
-        channel_identifier=event["args"]["channel_identifier"],
-        block_number=event["blockNumber"],
+        token_network_address=decode_hex(event["address"]), block_number=event["blockNumber"],
     )
```

Comment:
We could leave the common_infos unchanged if we move the `if event_name == "DeprecationSwitch"` before this. I don't see a downside and it keeps the code more compact.

Reply:
What do you think about the event itself, after sleeping over it I don't see any actual use, so I'd remove it.

Reply:
Yes, probably a good idea, too. We don't need to do anything on that event.